--- CoinMP/configure	2017-10-20 14:33:30.041582210 -0400
+++ CoinMP/configure	2017-10-20 14:38:04.082327789 -0400
@@ -2826,7 +2826,7 @@
           *-darwin*)
             ;;
           *)
-            coin_warn_cflags="-pedantic-errors $coin_warn_cflags"
+            coin_warn_cflags=" $coin_warn_cflags"
             ;;
         esac
     esac
@@ -3695,7 +3695,7 @@
           *-darwin*)
             ;;
           *)
-            coin_warn_cxxflags="-pedantic-errors $coin_warn_cxxflags"
+            coin_warn_cxxflags=" $coin_warn_cxxflags"
             ;;
         esac
     esac
